client/components/domains/domain-product-price/index.jsx

```diff
@@ -47,13 +52,29 @@ class DomainProductPrice extends React.Component {
 				break;
 			case 'INCLUDED_IN_HIGHER_PLAN':
 				if ( isEligibleVariantForDomainTest ) {
-					message = translate( 'Registration fee: {{del}}%(cost)s{{/del}} {{span}}Free{{/span}}', {
-						args: { cost: this.props.price },
-						components: {
-							del: <del />,
-							span: <span className="domain-product-price__free-price" />,
-						},
-					} );
+					if ( selectedPaidPlanInSwapFlow ) {
+						message = translate(
+							'Registration fee: {{del}}%(cost)s{{/del}} {{span}}Free with your plan{{/span}}',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Registration fee: {{del}}%(cost)s{{/del}} {{span}}Free with paid plan{{/span}}' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/swap-test-new-experiment-beta)